src/4_run_evaluation.py

```diff
@@ -452,6 +452,10 @@ def enumerate_tasks(self, max_instance_count: int):
                 # https://github.com/JHU-CLSP/turk-instructions/issues/66
```

Comment:
Did you mean to drop these lines?

Reply:
Yeah I meant to drop these lines since I thought they were polluting our logs when we were running our tests, made it hard to scroll all the way to the end and quickly see which task we were failing. I will convert these into logging.info right now though! good catch